eyeshade/workers/reports.js

```diff
@@ -1096,6 +1111,9 @@ exports.workers = {
       const summaryP = payload.summary
       const threshold = payload.threshold || 0
       const verified = payload.verified
+      const {
```

Comment:
can you change this to be consistent with the style of how other default values are handled?